libs/common/src/errors/portal-facility-amendment-conflict.error.ts

```diff
@@ -2,10 +2,10 @@ import { HttpStatusCode } from 'axios';
 import { ApiError } from './api.error';
 
 export class PortalFacilityAmendmentConflictError extends ApiError {
-  constructor(dealId?: string) {
-    const message = dealId
-      ? `There is already a portal facility amendment under way on deal ${dealId}`
-      : 'There is already a portal facility amendment under way on the deal';
+  constructor(facilityId?: string) {
+    const message = facilityId
+      ? `There is already a portal facility amendment under way on the facility ${facilityId}`
```

Comment:
@zainUKEF In my previous PRs I requested to change under way to be in progress. Can you updated here too to be consistent
```suggestion
      ? `There is already a portal facility amendment in progress on the facility ${facilityId}`
```

Reply:
`for the given facility...` please.